src/test/java/study/racingcar/RacingGameTest.java

```diff
@@ -4,56 +4,94 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import java.util.List;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class RacingGameTest {
-
-    SampleMovableDistance sampleMovableDistance;
+    private final String[] EXAMPLE_CARS = "pobi,crong,honux".split(",");
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private final String[] EXAMPLE_CARS = {"pobi", "crong", "honux"};
```